beacon_chain/state/state_transition.py

```diff
@@ -60,20 +74,30 @@ def get_crosslink_aggvote_msg(shard_id, shard_block_hash, crystallized_state):
         crystallized_state.last_justified_epoch.to_bytes(8, 'big')
 
 
-def get_attesters_and_signer(crystallized_state, active_state, skip_count):
-    attestation_count = min(len(crystallized_state.active_validators), ATTESTER_COUNT)
+def get_attesters_and_signer(crystallized_state,
```

Comment:
Should this be renamed to `get_attesters_and_proposer` to be aligned with the Casper v2 doc?

Reply:
yes